### ISSUE TITLE
Style/MethodCallWithoutArgsParentheses should have an IgnoredMethods setting

### ISSUE THREAD

**Issue by mvz** [bbatsov/rubocop#4753]
The Style/MethodCall**With**ArgsParentheses cop allows some methods to be ignored using the IgnoredMethods setting. Oddly, the Style/MethodCall**Without**ArgsParentheses cop does not.

--------

## Expected behavior

Given the following code

```ruby
s()
```

If I add the following to `.rubocop.yml`:

```yaml
Style/MethodCallWithoutArgsParentheses:
  IgnoredMethods: ['s']
```

I expect RuboCop not to report an offense for Style/MethodCallWithoutArgsParentheses.

## Actual behavior

RuboCop complains about the parameter and reports an offense for Style/MethodCallWithoutArgsParentheses:

```
Warning: unrecognized parameter Style/MethodCallWithoutArgsParentheses:IgnoredMethods found in /home/matijs/Projects/ripper_ruby_parser/.rubocop.yml
Inspecting 1 file
C

Offenses:

foo.rb:1:2: C: Do not use parentheses for method calls with no arguments.
s()
 ^
```

## Steps to reproduce the problem

Create a file `foo.rb` with:

```ruby
s()
```

And a file `.rubocop.yml` with:

```yaml
Style/MethodCallWithoutArgsParentheses:
  IgnoredMethods: ['s']
```

Run RuboCop:
```
rubocop foo.rb
```

## RuboCop version

```
$ rubocop -V
0.50.0 (using Parser 2.4.0.0, running on ruby 2.4.1 x86_64-linux)
```

**Comment by Drenmi** [comment]
Good point. 🙂 It's also a bit unfortunate the author went with `IgnoredMethods`, because I've been trying to find a good abstraction for `Whitelist`/`Blacklist`. 😅

**Comment by bbatsov** [comment]
> Good point. 🙂 It's also a bit unfortunate the author went with IgnoredMethods, because I've been trying to find a good abstraction for Whitelist/Blacklist. 😅

It's not like we can't change this. ;-)

**Comment by bbatsov** [comment]
There are unfortunately a ton of inconsistencies from the early days of the project.

**Comment by Drenmi** [comment]
Indeed. Another one I've noticed is the varied use of `Unnecessary`, `Useless`, `Unneeded`. (Probably influenced on how frustrated the author was at the offense at the time of writing. 😛)

**Comment by mvz** [comment]
Works like a charm, thanks!
